core/symbol/capitalize_spec.rb

```diff
@@ -21,6 +21,13 @@
     end
   end
 
+  ruby_version_is '2.4' do
+    it "capitalizes the first character regardless of if it is Unicode" do
+      "äöü".to_sym.capitalize.should == :"Äöü"
+      "aou".to_sym.capitalize.should == :"Aou"
```

Comment:
Could you use Symbol literals for the left part too? `"äöü".to_sym` => `:"äöü"`.
I think it would be more consistent.